### ISSUE TITLE
SHACL property shapes for action:actionStatus on action:ActionLifecycle are incorrect

### ISSUE THREAD

**Issue by sbarnum** [ucoProject/UCO#442]
# Bug description

The current SHACL property shape for action:actionStatus on action:ActionLifecycle is

```turtle
[
	sh:datatype vocabulary:ActionStatusTypeVocab ;
	sh:maxCount "0"^^xsd:integer ;
	sh:nodeKind sh:Literal ;
	sh:path action:actionStatus ;
]
```

This is the old style.
It should be the following like it is on action:Action

```turtle
[
	sh:datatype vocabulary:ActionStatusTypeVocab ;
	sh:message "Value is outside the default vocabulary ActionStatusTypeVocab." ;
	sh:path action:actionStatus ;
	sh:severity sh:Info ;
] ,
[
	sh:maxCount "1"^^xsd:integer ;
	sh:nodeKind sh:Literal ;
	sh:or (
		[
			sh:datatype vocabulary:ActionStatusTypeVocab ;
		]
		[
			sh:datatype xsd:string ;
		]
	) ;
	sh:path action:actionStatus ;
] ,
[
	sh:message "Value is not member of the vocabulary ActionStatusTypeVocab." ;
	sh:or (
		[
			sh:datatype vocabulary:ActionStatusTypeVocab ;
			sh:in vocabulary:ActionStatusTypeVocab-members ;
		]
		[
			sh:datatype xsd:string ;
		]
	) ;
	sh:path action:actionStatus ;
]
```

## Steps to reproduce

Any use of action:actionStatus on action:ActionLifecyle that is not a valid string member of ActionStatusTypeVocab should demonstrate the bug.

# Coordination

- Tracking in Jira ticket [OC-262](https://unifiedcyberontology.atlassian.net/browse/OC-262)
- [x] Administrative review completed
- [ ] Implementation has not been merged into `develop`

- [x] Milestone linked
- [ ] Documentation logged in pending release page

**Comment by kchason** [comment]
If I understand this bug properly, this looks to have already been implemented in [c22b4bae](https://github.com/ucoProject/UCO/commit/c22b4bae4eb7fff9440f8f8a0ab8e2dbc0d1eacd).

https://github.com/ucoProject/UCO/blob/master/ontology/action/action.ttl#L106

**Comment by ajnelson-nist** [comment]
> If I understand this bug properly, this looks to have already been implemented in [c22b4bae](https://github.com/ucoProject/UCO/commit/c22b4bae4eb7fff9440f8f8a0ab8e2dbc0d1eacd).
> 
> https://github.com/ucoProject/UCO/blob/master/ontology/action/action.ttl#L106

The *second* place this was supposed to have been used is here:
https://github.com/ucoProject/UCO/blob/master/ontology/action/action.ttl#L313

I'd missed it.  @sbarnum correctly caught that this was a miss.

**Comment by ajnelson-nist** [comment]
I just realized this bug is actually moot.

The shape in question has a `sh:maxCount 0` constraint.  So, all of the other constraints, including the one that seemed to induce the semi-open vocabulary problem, are irrelevant.

This would have been made more clear if the [SHIR update proposal](https://github.com/ucoProject/UCO/issues/417) had had a follow-on Issue enacted to remove redundant constraints.

I came across the "mootness" while implementing a resolution to [Issue 445](https://github.com/ucoProject/UCO/issues/445).
